packages/website/docs/autocomplete-js.md

```diff
@@ -56,21 +84,15 @@ const autocompleteSearch = autocomplete({
 });
 ```
 
-## Import
+## Parameters
 
-```ts
-import { autocomplete } from '@algolia/autocomplete-js';
-```
-
-## Params
-
-`autocomplete` accepts all the props that [`createAutocomplete`](/docs/createAutocomplete#reference) supports.
+The `autocomplete` function accepts all the props that [`createAutocomplete`](/docs/createAutocomplete#reference) supports.
 
 ### `container`
 
 > `string | HTMLElement` | **required**
 
-The container for the Autocomplete search box. You can either pass a [CSS selector](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors) or an [Element](https://developer.mozilla.org/docs/Web/API/HTMLElement). The first element matching the provided selector will be used as container.
+The container for the Autocomplete search input. You can either pass a [CSS selector](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors) or an [Element](https://developer.mozilla.org/docs/Web/API/HTMLElement). If there are several containers matching the selector, Autocomplete picks up the first one.
```

Comment:
This is more a full-featured search box than an input.

Reply:
It's even the whole experience, isn't it?

```suggestion
The container for the Autocomplete search experience. You can either pass a [CSS selector](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors) or an [Element](https://developer.mozilla.org/docs/Web/API/HTMLElement). If there are several containers matching the selector, Autocomplete picks up the first one.
```

Reply:
Not necessarily because the panel container is appended to the body by default.

Reply:
Got it. Changed in a207ec8e.